packages/server/test/complete.test.ts

```diff
@@ -292,7 +784,7 @@ test("conplete columns from duplicated alias", () => {
   expect(result.candidates.length).toEqual(11)
 })
 
-test("conplete columns innside function", () => {
+test("conplete columns innside function 1", () => {
```

Comment:
```suggestion
test("complete columns inside function 1", () => {
```

Can we name it something intentionally instead of "1" - "4"?

Reply:
Hey @joe-re, will do thanks for reviewing. I'll try to do this today. Cheers.